src/utils/balancerV2Contracts.ts

```diff
@@ -0,0 +1,92 @@
+export const balancerV2Contracts = {
+    Authorizer: {
+        '1': '0xA331D84eC860Bf466b4CdCcFb4aC09a1B43F3aE6',
+        '10': '0xA331D84eC860Bf466b4CdCcFb4aC09a1B43F3aE6',
+        '56': '0xA331D84eC860Bf466b4CdCcFb4aC09a1B43F3aE6',
+        '100': '0xA331D84eC860Bf466b4CdCcFb4aC09a1B43F3aE6',
```

Comment:
I usually find it a bit annoying to read through ids like these 😬 
Being able to read chain names during a sanity check on the addresses is a better experience.
If it's a small change in the script, I'd say it would be _nice to have_, but if it ends up being a pain or if it requires updating multiple files, I'd say it's ok as is.

Reply:
Was a bit tricky but definitely worth it :raised_hands: 

Make it alot easier to sanity check and see things like no `LBPPoolFactory` is deployed to Sonic yet

Reply:
Thanks for making the change! 🙏 
One last thing about them -> I see they are now sorted differently then before.
I'm just wondering if that order is random by any chance. If yes, it would mean that running the script to update addresses would generate a different order, which is not ideal for evaluating diffs.
I'm ok on sorting by id or alphabetically by chain name, but it would be great if that was consistent.